Px.Utils.TestingApp/Commands/DataValidationBenchmark.cs

```diff
@@ -53,17 +53,17 @@ private void ValidateDataBenchmarks()
         {
             using Stream stream = new FileStream(TestFilePath, FileMode.Open, FileAccess.Read);
             stream.Position = start + dataKeyword.Length + readStartOffset; // skip the '=' and linechange
-            DataValidator validator = new(stream, expectedCols, expectedRows, TestFilePath, 0, encoding);
-            validator.Validate();
+            DataValidator validator = new(expectedCols, expectedRows, 0);
+            validator.Validate(stream, encoding, TestFilePath);
```

Comment:
Should the encoding of the file be an optional parameter? If left empty the validator could try to solve it.

Reply:
Yes, I think that's how it originally was, but I accidentally changed it during my later changes. I'll change that back.